src/main/kotlin/step2/StringCalculator.kt

```diff
@@ -0,0 +1,43 @@
+package step2
+
+import step2.domain.Calculator
+import java.util.StringTokenizer
+
+fun main() {
+    println("계산식을 입력해주세요.")
+    val input = readLine() ?: throw IllegalArgumentException("입력값이 없습니다.")
+    println(StringCalculator.calculate(input))
+}
+
+private const val INPUT_DELIMITER = " "
+
+class StringCalculator(input: String) {
+    init {
+        if (input.isBlank()) {
+            throw IllegalArgumentException("입력 값이 비어있습니다.")
+        }
+        calculate(input)
+    }
+
+    companion object {
+        fun calculate(s: String): Int {
+            val st = StringTokenizer(s, INPUT_DELIMITER)
```

Comment:
```suggestion
            val tokens = StringTokenizer(s, INPUT_DELIMITER)
```
이번 과정을 진행하면서는 약어를 사용하기 보다는 변수명을 풀어서 선언해보세요. 
가독성이 좋아지는 장점이 있습니다 😄

Reply:
네, 조언해 주신대로 변수명을 풀어서 선언하겠습니다. `StringTokenizer`를 사용하지 않도록 수정하였고, `st` 변수는 삭제되었습니다. 😃 

- https://github.com/next-step/kotlin-racingcar/commit/ecc6214932b5660ef953d6df11e33974b49b5d12